cmd/minikube/cmd/start.go

```diff
@@ -142,8 +145,37 @@ func runStart(cmd *cobra.Command, args []string) {
 		glog.Errorln("Error getting VM IP address: ", err)
 		cmdUtil.MaybeReportErrorAndExit(err)
 	}
+
+	selectedKubernetesVersion := viper.GetString(kubernetesVersion)
+
+	// Load profile cluster config from file
+	cc, err := loadConfigFromFile(viper.GetString(cfg.MachineProfile))
+	if err != nil {
+		if os.IsNotExist(err) {
+			fmt.Println("No existing profile config found. A new profile config would be created.")
+		} else {
```

Comment:
Added this error check to avoid printing error message when the file doesn't exists.